docs/how_to/parameterization.md

```diff
@@ -182,7 +182,7 @@ The `replace_value` field in the `find_replace` parameter supports fabric-cicd d
     -   **Item attribute variable:** replaces the item's attribute value with the corresponding attribute value of the item in the deployed/target workspace.
         -   `$items.<item_type>.<item_name>.<attribute>` (legacy format)
         -   **`$items.<item_type>.<item_name>.$<attribute>`** (new format)
-        -   **Supported attributes**: `id` (item ID of the deployed item), `sqlendpoint` (sql connection string of the deployed item, only applicable to lakehouse and warehouse items), and `queryserviceuri` (query uri of the deployed item, only applicable to eventhouse item). Attributes should be lowercase.
+        -   **Supported attributes**: `id` (item ID of the deployed item), `sqlendpoint` (sql connection string of the deployed item, only applicable to lakehouse and warehouse items), `sqlendpointid` (sql endpoint ID of the deployed item, only applicable to lakehouse items), and `queryserviceuri` (query uri of the deployed item, only applicable to eventhouse item). Attributes should be lowercase.
```

Comment:
In this section, instead of providing information in a list format, can we create a chart view of the supported attributes, their definition and the applicable item types. For example: id , the item ID of the deployed item, any item type